lib/commons/forms/is-native-textbox.js

```diff
@@ -15,14 +18,16 @@ const nonTextInputTypes = [
  * Determines if an element is a native textbox element
  * @method isNativeTextbox
  * @memberof axe.commons.forms
- * @param {Element} node Node to determine if textbox
+ * @param {VirtualNode|Element} node Node to determine if textbox
  * @returns {Bool}
  */
 function isNativeTextbox(node) {
-	const nodeName = node.nodeName.toUpperCase();
+	node = node instanceof AbstractVirtuaNode ? node : getNodeFromTree(node);
+	const nodeName = node.props.nodeName;
 	return (
-		nodeName === 'TEXTAREA' ||
-		(nodeName === 'INPUT' && !nonTextInputTypes.includes(node.type))
+		nodeName === 'textarea' ||
+		(nodeName === 'input' &&
+			!nonTextInputTypes.includes((node.attr('type') || '').toLowerCase()))
```

Comment:
May not be necessary for this PR, but I think we should consider adding `virtualNode.props.type` by default, in the same way that we set nodeType. That way we don't have to do this type casting, and it'd give us an easier way to deal with inputs defaulting to type=text.